portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+jQuery(document).ready(function($) {
+
+
+	var mastheadheight = $('.ds-header').outerHeight();
+    console.log(mastheadheight);
+	$(".ds-banner,.ds-main-section").css("margin-top" , mastheadheight);
+
+	$(window).scroll(function(){
+	    if ($(window).scrollTop() >= 10) {
+	        $('.ds-header').addClass('ds-fixed-header');
+	    }
+	    else {
+	        $('.ds-header').removeClass('ds-fixed-header');
+	    }
+	}).scroll();
+
+
+ });
+
+ /**
+  * fetches the string from servlet
+  */
+
+  async function getString(){
+      const responseFromServer= await fetch('/hello');
+      const textFromResponse= await responseFromServer.json();
+     // console.log(textFromResponse);
+      const quote = textFromResponse[Math.floor(Math.random() * textFromResponse.length)];
+      const quoteContainer = document.getElementById('quote-container');
+      quoteContainer.innerText = quote;
+  }
+
+
+
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
+function addRandomQuote() {
   const quotes =
-      ['I am unbeatable in Othello', 'It’s the little things in life', 'Fake it till you make it' , 'I have my second toe bigger than my first toe', 'I can touch my nose with my tongue'];
+      ['I have never lost a game of Othello.', 'I was born in Delhi, India.', 'My second toe is bigger than my first toe.', 'I can touch my nose with my tongue.'];
```

Comment:
This is something for a follow-up PR, but if you're fetching all quotes in `getString`, you can probably save those and just replace which element is being shown. That way you also don't need to update the list in two places.

Reply:
I removed the addRandomQuote method and now the getString fetches all the strings and generates a random string. But when I open the webpage I see the string already even when the button is not pressed and I am not sure why. All I want is to display the random string only when the button is pressed.

Reply:
I can't see your updated code, but could it be that your function is still mentioned under `<body onload="getString()">`? That will trigger as soon as the website is shown.